### ISSUE TITLE
[Bug]: Tap Dial Switch not working

### ISSUE THREAD

**Issue by sammachin** [mauricedominic/node-red-contrib-huemagic#23]
### Node-Red-Contrib-Huemagic-Fork Version

4.2.8

### Node JS Version

18.19.0

### NPM Version

10.2.3

### Node-Red Version

3.1.3

### Operating System

Raspbian GNU/Linux 10 (buster)

### What happened?

Just added a new Hue Tap Dial switch to my system, the dial works fine when configured in the Hue app to control a light but it doesn't send anything to the Node-RED nodes, either when paired with a light or not.
Tried both the Hue Buttons and the Hue Dial nodes, no output on either and clicking the button in the node just returns `The button/switch in not yet available. Please wait until HueMagic has established a connection with the bridge or check whether the resource ID in the configuration is valid.`

### How to reproduce?

Add Hue Button and Hue Dial nodes, connected to debug nodes

### Expected behavior:

Expect output from nodes

### Additional comments?

Have tried resetting the tap dial and re-pairing it, to no avail.
I have several other hue buttons all working fine.

Not sure if there might be a software update for the tap dial required, or how to check?

**Comment by sammachin** [comment]
found the software update and updated the tap dial to `RDM002 | 2.59.19 | 100b-121` however same problem

**Comment by sammachin** [comment]
@rainbow-six3 sorry to bother you but wondered if you had any ideas as I saw the tap dial patch came from you?

**Comment by sammachin** [comment]
UPDATE:

I started to try and debug this myself, I added a console.log to the dial.js code in the `bridge.subscribe` event to see what the `currentState` object was from the bridge.get, that was printing out what looked like the correct device response.

I then added the events package (https://flows.nodered.org/node/@yadomi/node-red-contrib-philipshue-events)to another node-red instance to see if I could see the raw eventsource data for the dial coming from the bridge and that was outputting the correct dial rotation events.

When I went back to my huemagic instance the dial is now sending events as is the button! 
So not really sure what fixed it, perhaps something wasn't being properly initialised in the huemagic nodes but the other debig activated it??

Will close this issue for now but might be usful if someone else sees this problem in future or if anyone can test with a brand new device.
